### PR TITLE
Manage contingency lists in the grid explore app

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/utils/equipment-types.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
```
Comment (AbdelHedhili) [inline src/utils/equipment-types.js]: 2021

File: src/utils/script-types.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
```
Comment (AbdelHedhili) [inline src/utils/script-types.js]: 2021

File: src/components/filters-editor.js [hunk 3]
```diff
@@ -0,0 +1,276 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
```
Comment (AbdelHedhili) [inline src/components/filters-editor.js]: 2021

**Review by AbdelHedhili — APPROVED** [review]
just some outdated license headers

File: src/utils/equipment-types.js (hunk 1 above)
Comment (FranckLecuyer) [reply src/utils/equipment-types.js]: Corrected

File: src/utils/script-types.js (hunk 2 above)
Comment (FranckLecuyer) [reply src/utils/script-types.js]: Corrected

File: src/components/filters-editor.js (hunk 3 above)
Comment (FranckLecuyer) [reply src/components/filters-editor.js]: Corrected

**Pull request merged**